token/program-2022/src/extension.rs

```diff
@@ -453,18 +458,31 @@ impl Extension for MintPaddingTest {
     const TYPE: ExtensionType = ExtensionType::MintPaddingTest;
     const ACCOUNT_TYPE: AccountType = AccountType::Mint;
 }
+/// Account version of the MintPadding
+#[cfg(test)]
+#[repr(C)]
+#[derive(Clone, Copy, Debug, PartialEq, Pod, Zeroable)]
+pub struct AccountPaddingTest(MintPaddingTest);
+#[cfg(test)]
+impl Extension for AccountPaddingTest {
+    const TYPE: ExtensionType = ExtensionType::AccountPaddingTest;
+    const ACCOUNT_TYPE: AccountType = AccountType::Account;
+}
 
 #[cfg(test)]
 mod test {
     use {
         super::*,
-        crate::state::test::{TEST_MINT, TEST_MINT_SLICE},
+        crate::state::test::{TEST_ACCOUNT, TEST_ACCOUNT_SLICE, TEST_MINT, TEST_MINT_SLICE},
         solana_program::pubkey::Pubkey,
     };
 
     #[test]
     fn mint_with_extension_pack_unpack() {
```

Comment:
There are a couple additional things that I was hoping to exemplify in a multi-extension test:
- That extensions can be initialized in any order, and unpacking still works
- Given an opaque data buffer, how to determine which extension(s) are present
This test is getting pretty long, though. Second test?

Reply:
Good ideas -- I added some tests which should cover this.  I also added a check on the padding for Account::LEN - Mint::LEN to make sure it's all zeroes.  Not sure if it's useful, but it feels safer